# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: target/mnemosyne/pkg_patch/openser.patch
# Copyright (C) 2002 - 2006 Alejandro Mery
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---


options at openser.cfg overwrite any option given by command line, and we don't like
that behaviour

--- ./etc/openser.cfg.orig	2006-04-02 21:43:47.000000000 +0200
+++ ./etc/openser.cfg	2006-04-02 21:45:06.000000000 +0200
@@ -6,20 +6,7 @@
 
 # ----------- global configuration parameters ------------------------
 
-debug=3            # debug level (cmd line: -dddddddddd)
-fork=yes
-log_stderror=no    # (cmd line: -E)
-
-/* Uncomment these lines to enter debugging mode 
-fork=no
-log_stderror=yes
-*/
-
-check_via=no	# (cmd. line: -v)
-dns=no          # (cmd. line: -r)
-rev_dns=no      # (cmd. line: -R)
 port=5060
-children=4
 fifo="/tmp/openser_fifo"
 
 #
